test/pass/simple-soversion.cmake

```diff
@@ -12,13 +12,13 @@ install_dir(${TEST_DIR}/libsimple CMAKE_ARGS -DBUILD_SHARED_LIBS=On -DCMAKE_WIND
 if(NOT WIN32 AND NOT APPLE)
     test_expect_file(${PREFIX}/lib/libsimple.so)
     test_expect_file(${PREFIX}/lib/libsimple.so.1)
-    test_expect_file(${PREFIX}/lib/libsimple.so.1.1.2)
+    test_expect_file(${PREFIX}/lib/libsimple.so.1.1.0)
```

Comment:
These tests shouldn't change if the `ROCM_LIBPATCH_VERSION` env variable is not set.

Reply:
As per the code the behavior is if ROCM_LIBPATCH_VERSION is not set, it defaults the patch version to 0 and appends ".0" to the so file name. So suggestions is to not added any patch version is it?

Reply:
>  So suggestions is to not added any patch version is it?

Nope, the so version should be exactly what is specified by `rocm_set_soversion` function. If `ROCM_LIBPATCH_VERSION` is defined then that should append the version defined by `rocm_set_soversion` with `.${ROCM_LIBPATCH_VERSION}`.

Reply:
I have updated the latest commit as per the above comment.